utils/scarb-ui/src/args/verbosity.rs

```diff
@@ -63,21 +74,31 @@ impl VerbositySpec {
     }
 
     fn integer_verbosity(&self) -> i8 {
-        let int_level = (self.verbose as i8) - (self.quiet as i8);
-        if self.is_present() {
-            int_level
+        if self.no_warnings {
+            -1
         } else {
-            self.verbosity
-                .map(Verbosity::level_value)
-                .unwrap_or(int_level)
+            let int_level = if self.no_warnings {
+                -1
```

Comment:
Why check twice? 🤔 
Can we do 
``` 
if self.no_warnings {
    return -1;
}
``` 
at the beginning of this function, and avoid changing anything else?

Reply:
Yep, I initially added the check at the beginning of the function, but later thought setting `int_level` first is just more in line with existing approach. Removed the extra check 👍🏻